app/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -20,7 +20,7 @@ export const WelcomeScreen = observer(function WelcomeScreen() {
   return (
     <View style={{ flex: 1, backgroundColor: "white" }}>
       <Screen preset="fixed" backgroundColor={color.transparent}>
-        <View style={{ flexDirection: "row" }}>
+        <View style={{ flexDirection: "row", marginTop: 25 }}>
           <Image source={Logo} />
           <View style={{ flex: 1 }} />
           <Image source={Hamburger} />
```

Comment:
I realized this Hamburger icon doesn't do anything - should we remove it?

Reply:
Yes let's remove it!